simplesat/sat/policy/undetermined_clause_policy.py

```diff
@@ -58,6 +58,11 @@ def _build_id_to_clauses(self, clauses):
         for c in clauses:
             for l in c.lits:
                 table[abs(l)].append(c)
+
+        # Make sure all installed packages appear in the table
+        for pid in self._installed_ids:
+            table[pid]
```

Comment:
Is the intent to make sure every `pid` is in the table ? If so, I would prefer dealing with the error explicitly (something like `if pid not in table: raise SomeMeaningfulError(...)`).